src/Nox.Generator/Application/Dto/EntityCreateDto.template.cs

```diff
@@ -20,7 +20,7 @@ namespace {{codeGeneratorState.ApplicationNameSpace }}.Dto;
 public partial class {{className}} : IEntityCreateDto<{{entity.Name}}>
 {
 {{- for key in entity.Keys }}
-    {{- if key.Type == "Nuid" || key.Type == "AutoNumber" || key.Type == "DatabaseGuid" -}}
+    {{- if key.Type == "Nuid" || key.Type == "AutoNumber" -}}
     {{ continue; -}}
     {{- end }}
     /// <summary>
```

Comment:
@gjorgji-grgovski-authoritypartners  if the key is Guid, we have agreed that should not be required?

Reply:
Desired changes were applied